common/presenters/move-to-meta-list-component.test.js

```diff
@@ -164,105 +161,16 @@ describe('Presenters', function() {
       })
     })
 
-    context('when date is today', function() {
-      let transformedResponse
-
-      beforeEach(function() {
-        this.clock = sinon.useFakeTimers(new Date(mockMove.date).getTime())
-
-        transformedResponse = moveToMetaListComponent(mockMove)
-      })
-
-      afterEach(function() {
-        this.clock.restore()
-      })
-
-      describe('response', function() {
-        it('should contain today in date value', function() {
-          const item = transformedResponse.items[2]
-
-          expect(item).to.deep.equal({
-            key: { text: '__translated__' },
-            value: { text: 'Sunday 9 Jun 2019 (Today)' },
-            action: undefined,
-          })
-        })
-      })
-    })
-
-    context('when date is yesterday', function() {
-      let transformedResponse
-
-      beforeEach(function() {
-        this.clock = sinon.useFakeTimers(
-          subDays(new Date(mockMove.date), 1).getTime()
-        )
-
-        transformedResponse = moveToMetaListComponent(mockMove)
-      })
-
-      afterEach(function() {
-        this.clock.restore()
-      })
-
-      describe('response', function() {
-        it('should contain tomorrow in date value', function() {
-          const item = transformedResponse.items[2]
-
-          expect(item).to.deep.equal({
-            key: { text: '__translated__' },
-            value: { text: 'Sunday 9 Jun 2019 (Tomorrow)' },
-            action: undefined,
-          })
-        })
-      })
-    })
-
-    context('when date is tomorrow', function() {
-      let transformedResponse
-
-      beforeEach(function() {
-        this.clock = sinon.useFakeTimers(
-          addDays(new Date(mockMove.date), 1).getTime()
-        )
-
-        transformedResponse = moveToMetaListComponent(mockMove)
-      })
-
-      afterEach(function() {
-        this.clock.restore()
-      })
-
-      describe('response', function() {
-        it('should contain yesterday in date value', function() {
-          const item = transformedResponse.items[2]
-
-          expect(item).to.deep.equal({
-            key: { text: '__translated__' },
-            value: { text: 'Sunday 9 Jun 2019 (Yesterday)' },
-            action: undefined,
-          })
-        })
-      })
-    })
-
     context('with only `date from`', function() {
-      const presentDate = '2010-04-14'
       const mockMove = {
         date_from: '2020-05-01',
       }
       let transformedResponse
 
       beforeEach(function() {
-        this.clock = sinon.useFakeTimers(new Date(presentDate).getTime())
-        sinon.stub(filters, 'formatDateWithDay').returnsArg(0)
         transformedResponse = moveToMetaListComponent(mockMove)
       })
 
-      afterEach(function() {
-        this.clock.restore()
-      })
-
       describe('response', function() {
         it('should render `date from` row', function() {
```

Comment:
Oh, we are using the backticks to highlight the name of a property? Where does it work?

Reply:
I think it's actually just something I've used when I've remembered to try and show that the assertion is checking for a particular thing. Not sure it has any actual effect in mocha though.